RecoEgamma/EgammaTools/python/regressionModifier_cfi.py

```diff
@@ -55,3 +142,11 @@
                                           uncertaintyKey_50ns = cms.vstring('gedphoton_EBUncertainty_50ns', 'gedphoton_EEUncertainty_50ns'),
                                           )
               )
+
+#by default we use the regression inappropriate to the main purpose of this release
+#life is simplier that way
+regressionModifier = regressionModifier94X.clone()
```

Comment:
I thought that the EGRegressionModifierV3 can be used by all, that it was just a matter of having the right payload.
An earlier iteration of this PR was using the old payloads with EGRegressionModifierV3 and that gave the same electrons (and with the same intent but no fix, photons ).

Reply:
originally yes but I ended up deciding against it. It went through a few iterations, one where it lost the functionality but I think its back now. Anyways I dropped the idea as its not needed.